specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/package.json

```diff
@@ -279,7 +279,7 @@
         "tags": [
           "Package"
         ],
-        "operationId": "Packages_ListByRuntimeEnvironment",
+        "operationId": "Package_ListByRuntimeEnvironment",
```

Comment:
Changing `operationId` is a breaking change for SDK, has this version been released for the SDKs?

Reply:
this feature is not released yet not even public preview, SDK has not been released.